Description: Disable a test requiring compiled test binaries
Last-Update: 2023-02-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -428,7 +428,7 @@
         assert_eq!(b"hello", &*output);
     }
 
-    #[test]
+    //#[test]
     fn test_parent_handles() {
         // This test invokes the `swap` test program, which uses parent_stdout() and
         // parent_stderr() to swap the outputs for another child that it spawns.
